org.blackcoinmore.blackmore-qt.metainfo.xml

```diff
@@ -31,7 +31,7 @@
   <releases>
     <release date="2023-07-04" version="13.2.1"/>
   </releases>
-  <content_rating type="oars-1.1"/>
+  <content_rating type="oars-1.0"/>
```

Comment:
this is not what I meant. 1.1 is the right version

Reply:
what did you mean?